package.json

```diff
@@ -46,12 +46,13 @@
     ],
     "testPathIgnorePatterns": [
       "/node_modules/",
+      "<rootDir>/integration-tests/create-react-app",
       "<rootDir>/packages/react-percy-webpack/src/entry/__tests__/test-project/"
     ]
   },
   "lint-staged": {
     "*.js": [
-      "eslint --fix --color --no-ignore",
```

Comment:
^ exactly that, if `--no-ignore` is not given, then the `.storybook` folder will be ignored, which has a bit of JS. Storybook does support custom config dirs though, so we could move it into `my-config-dir` (w/o) dot and use `build-storybook  --config-dir=my-config-dir` if that is preferable.